src/api.ts

```diff
@@ -1,9 +1,25 @@
 import { debug } from '@actions/core';
 import { HttpClient } from '@actions/http-client';
 import { BearerCredentialHandler } from '@actions/http-client/lib/auth';
-import { RunResponse } from './main';
 import { GithubMetadata } from './fetch-metadata';
 
+export enum TestStatus {
+  PASSED = 'PASSED',
+  FAILED = 'FAILED',
+  RUNNING = 'RUNNING',
+  ERROR = 'ERROR',
+  FLAKY = 'FLAKY',
+  CANCELLED = 'CANCELLED',
+  SKIPPED = 'SKIPPED'
+}
```

Comment:
Typescript enums aren't great , let's not use them.

```suggestion
export type TestStatus = 'PASSED' | 'FAILED' | 'RUNNING' | 'ERROR' | 'FLAKY' | 'CANCELLED' | 'SKIPPED';
```